docs/find-lost-trn-integration.md

```diff
@@ -0,0 +1,72 @@
+# Find a lost TRN integration with the authorization server
+
+When the `qualified_teacher` scope is requested by a client, the authorization server needs to integrate with the
```

Comment:
@aje54 I've called the scope `qualified_teacher` in the spike as `dqt` seemed a little too implementation-specific.

Reply:
quailified_teacher scope should be where the record has QTS - is that what you mean? Otherwise we had spoken about a "trn_scope" which was the generic "find the record in DQT" scope that doesn't really care if they have QTS.

Reply:
That makes sense. Sounds like `trn` is what we want here.

Reply:
as per slack convo, use "trn" scope or similar